docs/servicebindings.md

```diff
@@ -0,0 +1,123 @@
+# Service Bindings
+
+kpack Images can be configured with Service Bindings as described in the [Kubernetes Service Bindings specification](https://github.com/k8s-service-bindings/spec).
+
+At build-time, service bindings are handled by the buildpacks being used for that Image. Check the desired buildpack documentation for details on the service bindings it supports.
+
+There are two ways to configure service bindings:
+
+1. Natively using a Secret (most common use case for kpack users)
+2. Using ProvisionedServices as described in the [specification](https://github.com/k8s-service-bindings/spec#provisioned-service) (requires additional operator configuration external to kpack)
+
+## Create a Service Binding with a Secret
+
+Requirements:
+
+* A Secret containing the service binding data
+  * The Secret `stringData` field **must** contain a key-value pairs of `type:<binding type>`. The buildpacks will use read this type
+  * The Secret `type` (not `stringData.type`) **must** be set to `service.binding/<binding type>` where `<binding type>` is the value of the key set in the above bullet.
```

Comment:
We don't validate this and I don't believe it is a hard requirement in the spec. Should we drop it to a **should**?